examples/rpl_udp/rpl.c

```diff
@@ -96,42 +96,46 @@ void rpl_udp_init(int argc, char **argv)
 
         printf("Channel set to %" PRIi32 "\n", chan);
 
+        /* global address */
+        ipv6_addr_t global_addr, global_prefix;
+        ipv6_addr_init(&global_prefix, 0xabcd, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0);
```

Comment:
Maybe a Unique Local IPv6 Unicast Address, in the range of fc00::/7 according to [RFC4193](http://tools.ietf.org/html/rfc4193) would be better.